examples/ct/handlers.go

```diff
@@ -225,40 +224,41 @@ func addChainInternal(c LogContext, w http.ResponseWriter, r *http.Request, isPr
 		return http.StatusMethodNotAllowed, fmt.Errorf("method not allowed: %s", r.Method)
 	}
 
-	// Check the contents of the request.
-	addChainRequest, err := parseBodyAsJSONChain(c, r)
+	// Check the contents of the request and convert to slice of certificates.
+	addChainReq, err := parseBodyAsJSONChain(c, r)
 	if err != nil {
 		return http.StatusBadRequest, err
 	}
-	validPath, err := verifyAddChain(c, addChainRequest, w, isPrecert)
+	chain, err := verifyAddChain(c, addChainReq, w, isPrecert)
 	if err != nil {
 		return http.StatusBadRequest, err
 	}
 
 	// Build up the SCT and MerkleTreeLeaf. The SCT will be returned to the client and
 	// the leaf will become part of the data sent to the backend.
-	merkleTreeLeaf, sct, err := signerFn(c.logKeyManager, validPath[0], c.timeSource.Now())
+	merkleLeaf, sct, err := signerFn(c.logKeyManager, chain[0], c.timeSource.Now())
```

Comment:
I'm generally sticking with a convention that protocol field names in structs (i.e. the things in `certificate-transparency/go/types.go`) match the RFC names wherever possible, but local vars have more Go-like terse names.   This one might even be better as just `leaf`...